webapp-shadcn/src/app/(auth)/forgot-password/page.tsx

```
@@ -0,0 +1,107 @@
+          <Link href="/login">
+            <Icons.arrowLeft className="mr-2 h-4 w-4" />
+            Back to login
+          </Link>
+        </Button>
+
+        <p className="text-center text-sm text-muted-foreground">
+          Didn&apos;t receive the email?{" "}
+          <button
+            onClick={() => setIsSubmitted(false)}
+            className="text-primary hover:underline"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)